.gitignore

```diff
@@ -3,6 +3,7 @@
 
 # Tool Runtime Dir
 /vNext/src/[Tt]ools/
+/vNext/src/[Bb]ootstrap/
```

Comment:
During the VS build, we output the bootstrap code generator to the Bootstrap folder so that all projects can access it.